docker/app/Dockerfile.prod

```diff
@@ -0,0 +1,25 @@
+FROM lf-app-base:prod
+
+# php customizations
+RUN mv $PHP_INI_DIR/php.ini-production $PHP_INI_DIR/php.ini
+COPY docker/app/customizations.php.ini $PHP_INI_DIR/conf.d/
+
+# apache2 customizations
```

Comment:
it's too bad there is a lot of duplication cut/paste between the two Dockerfiles and that there doesn't seem to be a supported way to INCLUDE another Dockerfile.  There's a long heated thread about people for/against (mostly for) an INCLUDE directive over at https://github.com/moby/moby/issues/735 and that led me to https://github.com/edrevo/dockerfile-plus which allows you to organize common commands into an include file so you can DRY.  I don't think anything is really actionable, just thought I would point out the duplication.  Not sure exactly how to minimize that with vanilla Dockerfile syntax.

Reply:
agreed on the disappointment, this is symptomatic of some of the coupling.  Also, I've never com across the need for `include` so I think it's probably just a matter of engineering things appropriately.